frontend/src/util.js

```diff
@@ -184,10 +185,6 @@ const addOrReplaceEdge = (graph, edge) => {
         if (!targetNode.data.predecessors.includes(sourceNode.id)) {
             targetNode.data.predecessors.push(sourceNode.id);
         }
-        targetNode.data.predecessorColumns = {
```

Comment:
then is setting predecessor columns always to [] even if it has columns, removed this block